src/components/UserActionFeedback/UserActionList.ts

```diff
@@ -0,0 +1,16 @@
+import cartIcon from "images/cart-empty.svg";
+
+export const UserAction = {
```

Comment:
Rename to PageFeedbackTypes
and create a typescript interface to type this object.

Reply:
Done